java/example/build.gradle

```diff
@@ -0,0 +1,5 @@
+plugins {
+    id 'java'
+}
+
+project.ext.setProperty("remotese_subproject_name", ":calypso:remotese")
```

Comment:
what is the reason of this property declaration?

Reply:
### tl;dr
Because the name of a project is relative and depend on the root of the gradle build call.

### Some more details
The idea is that any layer of the project is valid as an "autonomous" project with no dependencies to the parent project.

For instance, if i'm interested by the Calypso examples only, I can copy or build the `example/calypso` directory only.

In this specific case two child projects are included:
- _calypso.pc_, which defines a spotless gradle task for all its child projects (the pc use cases)
- _calypso.remotese_, which doesn't defines the spotless task to ease the use of a tester

That's why a spotless task is specifically configured for the remotese project, from the calypso layer. But, because it is specifically configured for the project, we need to specifically define the project name depending on where the gradle build is triggered from.

**Conclusion**: If it's triggered from the root, then the correct name is `:calypso:remotese`, if it's triggered from the calypso level, then the correct name is `:remotese`.